nautobot/circuits/tables.py

```diff
@@ -9,7 +9,31 @@
     TagColumn,
     ToggleColumn,
 )
-from .models import Circuit, CircuitType, Provider
+from .models import Circuit, CircuitType, Provider, ProviderNetwork
+
+CIRCUIT_TERMINATION_PARENT = """
+{% if value.provider_network %}
+    <a href="{{ value.provider_network.get_absolute_url }}">{{ value.provider_network }}</a>
+{% elif value.site %}
+    <a href="{{ value.site.get_absolute_url }}">{{ value.site }}</a>
+{% endif %}
```

Comment:
Seeing as the return value of `ProviderNetwork.get_absolute_url()` is already based on whether `ProviderNetwork.site` is set, this could be streamlined.
```suggestion
<a href="{{ value.provider_network.get_absolute_url }}">
{% if value.provider_network %}
    {{ value.provider_network }}
{% elif value.site %}
    {{ value.site }}
{% endif %}
</a>
```